import_data/import_from_dump.go

```diff
@@ -0,0 +1,57 @@
+package import_data
```

Comment:
Seems like this does both schema and data.  please name the package as import instead

Reply:
Done.